pmd-plsql/src/main/java/net/sourceforge/pmd/lang/plsql/symboltable/SymbolFacade.java

```diff
@@ -6,8 +6,13 @@
 
 import net.sourceforge.pmd.lang.plsql.ast.ASTInput;
 
-public class SymbolFacade {
-    public void initializeWith(ASTInput node) {
+public final class SymbolFacade {
```

Comment:
Should we mark that as well `@InternalApi`?

Reply:
I think so, but many other classes in this package should be internalized. The base symbol table classes should also be moved out of pmd-core into pmd PLSQL I think